ratpack-core/src/main/java/ratpack/server/internal/DefaultRatpackServer.java

```diff
@@ -319,7 +318,7 @@ public synchronized boolean isRunning() {
 
   @Override
   public synchronized String getScheme() {
-    return isRunning() ? sslEngine == null ? "http" : "https" : null;
+    return isRunning() ? !this.ssl ? "http" : "https" : null;
```

Comment:
in this case I think
`return isRunning() ? this.ssl ? "https" : "http" : null;` reads cleaner